core/txpool/blobpool/blobpool.go

```diff
@@ -1539,3 +1540,19 @@ func (p *BlobPool) Status(hash common.Hash) txpool.TxStatus {
 	}
 	return txpool.TxStatusUnknown
 }
+
+func (p *BlobPool) FilterBundle(bundle *types.Bundle) bool {
```

Comment:
It is a bit strange that blobpool implement such interface.

Reply:
Done